miniapps/plasma/cold_plasma_dielectric_solver.cpp

```diff
@@ -373,12 +370,20 @@ CPDSolver::CPDSolver(ParMesh & pmesh, int order, double omega,
    }
    if (kCoef_)
    {
+      if (L2FESpace_ == NULL)
+      {
+         L2FESpace_ = new L2_ParFESpace(pmesh_,order-1,pmesh_->Dimension());
+      }
```

Comment:
Here, `order-1` is used for the L2 space, but the L2 vector space below uses `order`. 
Should it both be `order-1`?